### ISSUE TITLE
StandaloneBuycraftRunner does not dispatch commands

### ISSUE THREAD

**Issue by toonsevrin** [BuycraftPlugin/BuycraftX#63]
Issue: 
When I create a simple standalonebuycraftrunner the verbose logs: 
`INFO: Fetched 2 commands for player 'toonsev'.`
but does not call the dispatcher method (and the player stays queued).

Replication:

``` java
 public static void main(String[] args){
        StandaloneBuycraftRunner runner = StandaloneBuycraftRunnerBuilder.builder()
                .apiKey("insert key")
                .dispatcher((str) -> System.out.println("dispatching: " + str))
                .determiner(getPlayerDeterminer())
                .executorService(new ScheduledThreadPoolExecutor(10))
                .logger(Logger.getLogger("buycraft webhook"))
                .verbose(true).build().start();
    }

    public static PlayerDeterminer getPlayerDeterminer(){
        return new PlayerDeterminer() {
            public boolean isPlayerOnline(QueuedPlayer queuedPlayer) {
                return true;
            }

            public int getFreeSlots(QueuedPlayer queuedPlayer) {
                return 20;
            }
        };
    }
```

**Comment by toonsevrin** [comment]
And I attached my API key hahah

**Comment by toonsevrin** [comment]
@minecrafter if you would like, we could go in a small skype session where you explain me the prospects of the Buycraft plugins. I'd love to contribute as I noticed you are planning to go inactive.

**Comment by minecrafter** [comment]
Please read the [relevant blog post](https://mcblog.minimum.io/goodbye-spigot/).

> **Buycraft**: I will continue developing the Buycraft plugin. No immediate changes are currently expected.

This is still true.

**Comment by toonsevrin** [comment]
Good to know, have you been able to track the issue above or shall I try to?

**Comment by toonsevrin** [comment]
Seems to be fixed when using #64

**Comment by liam-wiltshire** [comment]
This issue is being closed as an old issue. For all support requests, please email support@buycraft.net. If you believe there is a plugin bug, please open a new issue.
